### ISSUE TITLE
"require" is not defined

### ISSUE THREAD

**Issue by SweetSea-ButImNotSweet** [MrMeCoding/TETR-VERGE#7]
Affected: TETR-VERGE for Browser
There is no "require" statement in Chromium-browser, it's only being used by Node.js

Browser: Chrome 121.0.6167.140 (Official Build) (64-bit) (cohort: Stable)

**Comment by KyraCoding** [comment]
Which version of VERGE are you using?

**Comment by KyraCoding** [comment]
Also, there was a critical error affecting version 0.0.1 of VERGE. You'll need to manually update to 0.0.2 via the GitHub files, sorry for the inconvenience

**Comment by SweetSea-ButImNotSweet** [comment]
> Which version of VERGE are you using?

The latest one, I will try again if I find out you pushed any nrw commits

**Comment by SweetSea-ButImNotSweet** [comment]
Also it failed on the start of the extension...

**Comment by MrMeCoding** [comment]
> > Which version of VERGE are you using?
> 
> The latest one, I will try again if I find out you pushed any nrw commits

Yes, the extension crashed initially due to another error, which has been fixed.
I should remove the require code from the browser version however, but the error doesn't cause any issues as of now.
The current version of VERGE (0.0.2) should no longer crash TETR.IO

**Comment by KyraCoding** [comment]
Two errors caused this bug, both of which have been fixed.

The current version of VERGE (0.0.2) should run without errors, let me know if it works for you.

**Comment by SweetSea-ButImNotSweet** [comment]
> Two errors caused this bug, both of which have been fixed.
> 
> The current version of VERGE (0.0.2) should run without errors, let me know if it works for you.

Extension loaded, but look like it is stuck in "TETR VERGE IS LOADING..." (network in good condition, TETR.IO loads not too slow, weird)
Lemme check again in TETR.IO Desktop (reinstalling time, ~~I don't play TETR.IO in Desktop because, I don't really want to spend extra 200MB of a browser-like app~~ ok I think I should play in it instead of a regular browser)

Update: TETR VERGE in Desktop client works normal, something is weird with browser, lemme try again
Update2: Ok I'm pretty sure the browser one is having the loading issue

**Comment by KyraCoding** [comment]
Yes, I would like to apologize for not thoroughly testing the browser version of VERGE. We hope to resolve all issues with the browser version in 0.0.3. The current issue is due to insufficient storage in the browser.
